packages/codefixes/src/types.ts

```diff
@@ -29,8 +29,16 @@ export interface CodeHint {
   hint: string;
 }
 
+export interface CodeFixCollectionFilter {
```

Comment:
We have codefixes files now. Can be extended in the future for any user experience improvement purpose :)